spec/dmv_facility_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'spec_helper'
+
+RSpec.describe DmvFacility do
+  
+  describe "#exists" do
+    it "exists" do
+      
+      add_facility1 = DmvDataService.new.or_dmv_office_locations
+      # require 'pry'; binding.pry
+      expect(add_facility1).to be_a(Array)
```

Comment:
Keep in mind that you don't need to make assertions on what the data looks like BEFORE it goes into your DmvFacility.createFacilities method. We just want to assert on what comes out.